README.md

```diff
@@ -1 +1,16 @@
-# efgramsbergen.github.io
+## Egbert's experimental homepage
+*Hello, I am Egbert Gramsbergen. I studied physics and now I am a data engineer. What is a data engineer? Nobody knows, including me.*
+
+**My github handle:** efgramsbergen
+
+**What to do in my hometown Delft:** climb the main church tower, then reward yourself with a beer in one of the many cafes. And don't buy Delft Blue pottery, it's only for tourists and it's a ripoff.
+
+**Favorite book at the moment:** Getallen zijn je beste vrienden (numbers are your best friends).
+
+**Favorite cities:** Rome, New York (but this list may change depending on my mood. Today I think Copenhagen is quite nice but things may change when it is raining tomorrow)
+
+**Favorite formulas:** Maxwell's equations. I will wear them on a T-shirt tomorrow for everyone to admire.
+
+![Maxwell's equations)](http://www.maxwells-equations.com/maxwells-equations.gif)
+
+**Favorite cartoon character:** Spongebob. Really? Yes, really.
```

Comment:
if you edit your file and commit on this line, maybe this comment will be hidden. can you try ?